### DISCUSSION TITLE
Add ability to programatically download stats

### DISCUSSION THREAD

**Discussion by JonBulman** [netzero-labs/netzero#57] [Ideas]
Hi

Each month I download the Netzero stats for the month using the download button in the app.

It would be great to have REST APIs to enable this to be done programmatically for a defined period. An example can be found here from SolarEdge:https://developers.solaredge.com/docs/monitoring/13d262845566d-site-energy

Cheers
Jon

**Comment by zigam** [comment]
Hi,

We wrote a tool for bulk and incremental downloads of data:
https://github.com/netzero-labs/tesla-solar-download

Hope this helps,
Ziga
